src/extension/index.ts

```diff
@@ -151,23 +152,27 @@ export async function activate(context: ExtensionContext) {
 
     // API
     const api = {
-        async openLogs(logs: Uri[], _options: unknown, cancellationToken?: CancellationToken) {
+        async openLogs(logs: Uri[], cancellationToken?: CancellationToken) {
```

Comment:
Also a breaking change

Reply:
Should this be added to the .d.ts file?

Reply:
We would appreciate for it to be (we do pass in a cancellation token), but Chris probably has the most context on whether it _should_ be.